asciidoc-confluence-publisher-converter/src/test/java/org/sahli/asciidoc/confluence/publisher/converter/AsciidocConfluencePageTest.java

```diff
@@ -99,6 +101,36 @@ public void render_asciidocWithTopLevelHeaderAndMetaInformation_returnsConfluenc
     }
 
     @Test
+    public void render_asciidocWithTopLevelHeaderAndMetaInformationAndPageTitlePostProcessorConfigured_returnsConfluencePageWithPostProcessedPageTitleFromTitleMetaInformation() throws Exception {
+        // arrange
+        String adoc = ":title: Page title (meta)\n" +
```

Comment:
Why did you add `:title:` we are not supporting this anymore.

Reply:
Support for `:title:` is already provided by asciidoctor - according to my understanding, it cannot be disabled (`DocumentHeader.getDocumentTitle()` already respects the `:title:` meta information). The feature is just not documented in our documentation.